libbeat/docs/step-configure-credentials.asciidoc

```diff
@@ -0,0 +1,29 @@
+ifeval::["{beatname_lc}"!="winlogbeat"]
+. If you've secured Elasticsearch and Kibana, you need to pass credentials when
+you run the commands that set up and start {beatname_uc}. You can either pass
+credentials at the command line (as shown later in the getting started guide),
```

Comment:
Would be possible to add a link to the section where the command line is present?